tests-api/src/main/java/com/gradle/develocity/api/tests/BuildsQueryUtils.java

```diff
@@ -0,0 +1,25 @@
+package com.gradle.develocity.api.tests;
+
+import java.time.OffsetDateTime;
+
+final class BuildsQueryUtils {
+    private BuildsQueryUtils() {
+    }
+
+    static String projectNameEquals(String projectName) {
```

Comment:
💬 We can probably include more parameters, but it seemed like an overkill for this app. I just wanted to illustrate how one can combine multiple build criteria with the Tests API.